### ISSUE TITLE
[BUG][code-analyzer] Issues with output format and display

### ISSUE THREAD

**Issue by davidbrowaeys** [forcedotcom/code-analyzer#1892]
### Have you tried to resolve this issue yourself first?

- [x] I confirm I have gone through the above steps and still have an issue to report.

### Bug Description

I was reading an [article from Salesforce](https://developer.salesforce.com/blogs/2025/07/elevate-your-teams-code-quality-with-salesforce-code-analyzer-v5) that we could generate a html file. But looking at the cli it only support .json .csv. Is that a future enhancement ? 

Also when I run the command in my terminal, I don't see the name of the files in the output. Please refer to screenshot below. 

<img width="1350" height="775" alt="Image" src="https://github.com/user-attachments/assets/f2126682-d87c-4652-ba56-0f9712fbf3fe" />

### Output / Logs

```shell

```

### Steps To Reproduce

open terminal and run: 
sf code-analyzer rules --rule-selector pmd

### Expected Behavior

Display the output nicely

### Operating System

windows

### Salesforce CLI Version

@salesforce/cli/2.103.7 win32-x64 node-v22.19.0

### Code Analyzer Plugin (code-analyzer) Version

code analyzer 5.4.0

### Node Version

v22.19.0

### Urgency

Moderate
